ReviewerUI/src/js/components/Layout.js

```diff
@@ -0,0 +1,29 @@
+import React from "react/lib/React";
+
+import Header from "./Header";
+import SearchCriteriaForm from "./SearchCriteriaForm";
+import SearchResultsForm from "./SearchResultsForm";
+
+export default class Layout extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      client: "table-next",
```

Comment:
I don't know yet how to pass data between two child-components not using the state of parent one. Could you show some example?